specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Thanks for clarifying your intention - I missed that. The first problem is that most folks want sampling to happen from the manifests, which don't have ballot-by-ballot style info. We might be able to finesse that as noted in a separate conversation. But still, if we do sampling by contest, wouldn't we have disjoint samples in the same county for different contests? We want a single sample to cover all audited contests. CORLA2 has always framed it in terms of sampling by county, and maybe someday doing so by CVR to optimize finding contests that don't appear on all ballots, but that last part is tricky since we can't trust the CVRs.